megalinter/reporters/GitlabCommentReporter.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+"""
+GitHub Status reporter
+Post a GitHub status for each linter
+"""
+import logging
+from megalinter.utils_reporter import build_markdown_summary
+import os
+
+import gitlab
+from megalinter import Reporter, config
+
+
+class GitlabCommentReporter(Reporter):
+    name = "GITLAB_COMMENT"
+    scope = "mega-linter"
+
+    gitlab_api_url = "https://gitlab.com/api/v4/"
+    gitlab_server_url = "https://gitlab.com"
+
+    def manage_activation(self):
+        if config.get("GITLAB_COMMENT_REPORTER", "true") != "true":
+            self.is_active = False
+        elif (
+            config.get("POST_GITLAB_COMMENT", "true") == "true"
+        ):  # Legacy - true by default
+            self.is_active = True
+
+    def produce_report(self):
+        # Post comment on GitHub pull request
+        if config.get("CI_JOB_TOKEN", "") != "":
```

Comment:
Hmm, nice! This works automagically with the [CI_JOB_TOKEN](https://docs.gitlab.com/ee/ci/variables/predefined_variables.html)? No configuration needed, hugh? :thinking:

Reply:
that's the goal ^^